src/main/java/com/erigitic/commands/JobCommand.java

```diff
@@ -221,8 +223,28 @@ public CommandResult execute(CommandSource src, CommandContext args) throws Comm
                     TEJobSet jobSet = optSet.get();
                     Currency defaultCurrency = totalEconomy.getDefaultCurrency();
 
-                    for (TEActionReward actionReward : jobSet.getActionRewards()) {
-                        lines.add(Text.of(TextColors.GOLD, "[", jobManager.titleize(actionReward.getAction()), "] ", TextColors.GRAY, actionReward.getTargetID(), TextColors.GOLD, " (", actionReward.getExpReward(), " EXP) (", defaultCurrency.format(actionReward.getMoneyReward()), ")"));
+                    for (TEAction action : jobSet.getActions()) {
+                        EconomyService service = Sponge.getServiceManager().provideUnchecked(EconomyService.class);
+                        Optional<Currency> c = Optional.empty();
+                        Text listText;
+                        if (action.isIDTraited()) {
+                            // MC does not support '\t'
+                            String tab = new String(new char[action.getAction().length() + 2]).replace("\0", " ");
+                            List<Text> texts = new ArrayList<>(action.getRewards().size());
+                            action.getRewards().forEach((k, v) -> {
```

Comment:
New line before statement

This should be done throughout all of the changed files.